recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
     - bokeh >=1.0.0,!=2.0.0
     - cytoolz >=0.8.2
     - dask-core {{ version }}
-    - distributed >={{ version }}
+    - distributed {{ version }}
```

Comment:
Note that we're pinning the `distributed` version to be the same as the `dask` version (xref https://github.com/dask/community/issues/155)